docs/source/response_structure.rst

```diff
@@ -70,6 +70,30 @@ Energy Totals per Fuel Time Series
 .. literalinclude:: schemas/response/EnergyTotalsPerFuelTimeSeries.json5
    :language: javascript
 
+Energy End Uses
+~~~~~~~~~~~~~~~
+
+.. literalinclude:: schemas/response/EnergyEndUses.json5
+   :language: javascript
+
+Energy End Uses per Fuel Time Series
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. literalinclude:: schemas/response/EnergyEndUsesPerFuelTimeSeries.json5
+   :language: javascript
+
+================================== ==========================================================================================================
+Property                           Notes
+================================== ==========================================================================================================
+``heating``                        Excludes heat pump backup and fans/pumps
+``heatingFansPumps``               Includes supply fan (air distribution) or circulating pump (hydronic distribution or geothermal loop)
+``heatingHeatPumpBackup``          Excludes heat pump backup fans/pumps
+``heatingHeatPumpBackupFansPumps`` Includes supply fan (air distribution) or circulating pump (hydronic distribution) during heat pump backup
+``cooling``                        Excludes fans/pumps
+``coolingFansPumps``               Includes supply fan (air distribution) and circulating pump (geothermal loop)
+``pv``                             Negative value for any power produced
+================================== ==========================================================================================================
```

Comment:
I see other properties missing in this table. Are you planning to include them in this PR or in a later iteration?

Reply:
I have currently only included the properties have have notes, but can certainly add the full list for clarity/consistency!